src/Native/System.Native/pal_process.cpp

```diff
@@ -59,6 +65,9 @@ static_assert(PAL_PRIO_PROCESS == static_cast<int>(PRIO_PROCESS), "");
 static_assert(PAL_PRIO_PGRP == static_cast<int>(PRIO_PGRP), "");
 static_assert(PAL_PRIO_USER == static_cast<int>(PRIO_USER), "");
 
+// We make assumptions based on these being the same
+static_assert(sizeof(char) == sizeof(uint8_t), "");
```

Comment:
Yeah, I just put that in to protect us in case something really wonky happens...it would save a bunch of debugging time :)